src/App.js

```diff
@@ -49,17 +50,28 @@ class App extends Component {
     }
 
     findCountry = (countryName) => {
-      let chosenCountryObj = this.state.countryData.countries.find((country) => {
-        if(country.name.toLowerCase() === countryName) {
-          return country;
-        }
-      })
+
+        let chosenCountryObj = this.state.countryData.countries.find((country) => {
+          if (country.name.toLowerCase() === countryName) {
+            return country;
```

Comment:
Remember that `find` will automatically return the first match it comes across, so you don't have to return `country` here. Your callback within your `find` method should just be returning `true` or `false` - so this could be simplified a bit:

```
.find(country => {
  return (country.name.toLowerCase() === countryName)
}
```